.github/workflows/directoriesFilesChangePR.yml

```diff
@@ -57,3 +57,53 @@ jobs:
         with:
           name: output_changed_based_build_list
           path: ./**/output_*/
+  triggerGrinderTests:
+    name: Build
+    runs-on: ubuntu-latest
+    outputs:
+      test_targets_str: ${{ steps.format_test_targets_str.outputs.test_targets_str }}
+    env:
+        TARGET_LIST:  ${{ needs.getBuildLists.outputs.buildLists }}
+    needs: getBuildLists
+    if: ${{ needs.getBuildLists.outputs.buildEnv }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: format test targets str
+      id: format_test_targets_str
+      run: |
+        python3 ./.github/workflows/createTestList.py ${{ needs.getBuildLists.outputs.buildLists }}
```

Comment:
Looks like `buildLists` are passed in as arguments. However there is no code dealing with the arguments in `createTestList.py `.  `createTestList.py ` is actually using environment `TARGET_LIST` setting the target tests. My suggestion would be parsing the arguments instead of using environment variable.